components/Tabs/README.md

```diff
@@ -58,6 +58,7 @@ Tabs are bars of buttons used to navigate between groups of content.
 - [Extensions](#extensions)
   - [Color Theming](#color-theming)
   - [Typography Theming](#typography-theming)
+  - [Theming Extensions (Beta)](#theming-extensions-(beta))
```

Comment:
I believe [we have a bug in our docs-generation script](https://github.com/material-components/material-components-ios/issues/7068) where this href is incorrect. Let's manually replace `-(beta)` with `-beta` as suggested in that Issue.

Reply:
Addressed in 4e8cb3cc3ec6eb9a0b5536dac519b6b866791415